validator/testdata/transformed_feature_tests/server_side_rendering.html

```diff
@@ -35,6 +36,14 @@
     <i-amphtml-sizer style=display:block;padding-top:171.4370%;></i-amphtml-sizer>
   </amp-img>
   <!-- Valid -->
+  <amp-img src="image.png" height="100" width="300" layout="intrinsic" class="i-amphtml-layout-intrinsic i-amphtml-layout-size-defined" i-amphtml-layout="intrinsic">
+    <i-amphtml-sizer class="i-amphtml-sizer"><img alt="" aria-hidden="true" class="i-amphtml-intrinsic-sizer" role="presentation" src="data:image/svg+xml;charset=utf-8,<svg height=&quot;100&quot; width=&quot;300&quot; xmlns=&quot;http://www.w3.org/2000/svg&quot; version=&quot;1.1&quot;/>"></i-amphtml-sizer>
```

Comment:
If [this](https://stackoverflow.com/questions/18467982/are-svg-parameters-such-as-xmlns-and-version-needed) is accurate, then we should probably drop the xmlns and version attributes on the inline svg.

Reply:
Not sure about this part:

```
If however you serve your page as image/svg+xml or application/xhtml+xml or any other MIME type that causes the user agent to use an XML parser then the xmlns attributes are required. This was the only way to do things until recently so there is a lot of content served like this.
```

We could remove it for docs served from the cache, but not for ones served from the origin.

Reply:
My 2c is that if you are doing this and relying on the browser to sniff and correct content-type you already have a big problem. I don't think the AMP cache would even ingest documents with weird content types like this.

Reply:
`xmlns` seems to be required. The image won't be displayed if it's omitted.